_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,129 @@
+---
+name: video only element has transcript
+description: |
+  Non-streaming `video` elements without audio must have all visual information available in a transcript.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video doesn't contain audio.
+
+### Expectation 1
+
+A text transcript is available for the test target, either on the page or available through a link.
+
+### Expectation 2
+
+The text transcript (from expectation 1) contains all the visual information of the test target.
```

Comment:
"from expectation 1" seems a little procedural. Think that expectation 1 and 2 should be combined into one.